src/js/components/Icon/Icon.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+const svgIcon = (name) => require(`../../icons/${name}`)
+
+// TODO: Icons flash 'ginormous' on page load, before css kicks in and adjust
```

Comment:
A solution to this is to set `width` and `height` attributes on the source SVG, preventing them from filling the screen until the CSS loads, and once it does the `.Icon` class overrides these properties. I thought I had includes those attributes already, but I guess not! I can get those attributes added.

Reply:
Cool, if I messed something up in the component or need to add styles, just let me know.,